datalad/core/distributed/tests/test_clone.py

```diff
@@ -487,15 +487,37 @@ def test_cfg_originorigin(path):
 
 
 # test fix for gh-2601/gh-3538
+@known_failure
```

Comment:
With your changes, this `@known_failure` should be dropped.  (I tried to push that change, but like you I'm [seeing permission issues][0] when trying to push to other people's branches.)

[0]: https://github.com/datalad/datalad/pull/4026#issuecomment-574729391

Reply:
True, indeed. Thx.